scalableminds/webknossos
# Add slack telemetry messages, remove NewRelic

### Steps to test:
- set slack hook url in application.conf (ask me for a valid url ;) )
- destroy an sql query (or your database)
- provoke wk to run that query
- we should see a notification in slack

### Issues:
- fixes #4186 

------
